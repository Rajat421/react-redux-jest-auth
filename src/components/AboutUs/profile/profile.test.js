/**
 * Created by Counter on 7/2/2017.
 */
import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'
import Profile from './profile'

test('Profile component should render as expected',()=>{
    const component =shallow(<Profile/>);
    const tree =toJson(component);
    expect(tree).toMatchSnapshot()
});