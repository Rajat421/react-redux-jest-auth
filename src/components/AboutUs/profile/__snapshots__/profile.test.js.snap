// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile component should render as expected 1`] = `
<div>
  <section
    className="aboutus section_padding"
  >
    <h3
      className="heading_h3"
    >
      Profile
    </h3>
    <p
      className="text_paragraphs"
    >
      Founded in 2004 by first generation entrepreneurs, Inteliment is an an Innovative Company that effectively syndicates the ‘Data Science & Art’ into a Multi-Disciplinary Analytics to deliver. Insight, Integrity and Impact! Inteliment has been on the path of synchronized value co-creation and innovations for some of the most powerful and impact-making companies in the world. Inteliment’s key differentiator is in its - deep Technical Competency, Customer Centricity a Proven Track Record of helping companies truly harnessing the power of their data for effective business decisions creating competitive advantage and success. Inteliment operates ISO 9001:2015 Certified India Development Centre and Innovation Labs in Pune, with a state-of-the-art facility adhering to the Global Quality Standards and international offices in Sydney and Singapore, servicing .80+ Global Customers across diver industry domains.
    </p>
  </section>
</div>
`;
